x-pack/test/reporting/functional/lib/compare_pdfs.js

```diff
@@ -44,6 +44,7 @@ export async function checkIfPdfsMatch(actualPdfPath, baselinePdfPath, screensho
   fs.writeFileSync(actualCopyPath, fs.readFileSync(actualPdfPath));
 
   const convertOptions = {
+    // '-density': '300',
```

Comment:
I think maybe try to bring this back?  Or get rid of?  I should have left a comment for why this was here.  I think it was because it helped with the resolutions of the baseline vs jenkins.  But maybe bc the threshold is so high right now it doesn't even matter.  I commented it out in that initial PR you copied from just because I thought maybe it would fix the imagemagic problem, but as you've discovered, that wasn't actually the culprit.